brainglobe_registration/widgets/adjust_moving_image_view.py

```diff
@@ -71,13 +86,22 @@ def __init__(self, parent=None):
             self._on_adjust_image
         )
 
-        self.adjust_moving_image_reset_button = QPushButton(parent=self)
+        self.adjust_moving_image_reset_button = QPushButton()
         self.adjust_moving_image_reset_button.setText("Reset Image")
         self.adjust_moving_image_reset_button.clicked.connect(
             self._on_reset_image_button_click
         )
 
-        self.layout().addRow(QLabel("Adjust the moving image: "))
+        self.layout().addRow(QLabel("Adjust the moving image scale:"))
+        self.layout().addRow(
+            "Sample image X pixel size:", self.adjust_moving_image_voxel_size_x
```

Comment:
```suggestion
            "Sample image X pixel size [\u03BCm]:", self.adjust_moving_image_voxel_size_x
```
Worth specifying micron as unit for clarity?

Reply:
Definitely, great suggestion!